src/parser.rs

```diff
@@ -156,21 +156,23 @@ where
     /// Inserts a new custom block into the parser
     pub fn block<B: Into<Box<dyn compiler::ParseBlock>>>(mut self, block: B) -> Self {
         let block = block.into();
-        self.blocks.register(block.reflection().start_tag(), block);
+        self.blocks
+            .register(&block.reflection().start_tag().to_owned(), block);
```

Comment:
So we're converting `&str` (start_tag) -> `String` (to_owned) -> `&str` (&) -> `String` (register)?

Reply:
True, I'll change register to String (and modify a bunch of tests)